editoast/src/views/infra/railjson.rs

```diff
@@ -161,6 +171,17 @@ async fn post_railjson(
     }))
 }
 
+fn invalid_speed_sections(speed_sections: &[SpeedSection]) -> Vec<String> {
+    let mut errors = Vec::new();
+    for section in speed_sections {
+        if let Err(_e) = section.validate() {
+            let error_message = format!("Validation error on speedsection {}", section.get_id());
+            errors.push(error_message);
+        }
+    }
+    errors
+}
```

Comment:
We should implement the validation trait to `Railjson`. Doing so could allow us to simply do something like:

```rust
let railjson = railjson.into_inner().validate()?;
```